lawlibrary/static/stylesheets/lawlibrary.scss

```diff
@@ -9,3 +9,26 @@ footer {
 .btn.btn-primary {
   color: #fff;
 }
+
```

Comment:
All the views and templates will be used by other LIIWeb websites, so need to be in the liiweb app, not lawlibrary.